Pod/Native/ParseState.swift

```diff
@@ -91,7 +90,7 @@ class ParseState {
                 
                 // The scenario description must be unique
                 let description = "\(description)_line\(example.lineNumber)"
-                scenarios.append(NativeScenario(description, steps: steps))
+                scenarios.append(NativeScenario(description, steps: steps, index: index + exampleIndex))
```

Comment:
Blast, yes, I got my example the wrong way round - sorry! But as long as inserting examples in a scenario affects the index of the next scenario then yea, this isn't an issue :)